react/features/base/lib-jitsi-meet/actions.js

```diff
@@ -44,19 +45,20 @@ export function initLib() {
             throw new Error('Cannot init lib-jitsi-meet without config');
         }
 
-        // FIXME Until the logic of conference.js is rewritten into the React
-        // app we, JitsiMeetJS.init is to not be used for the React app.
-        if (typeof APP !== 'undefined') {
-            return Promise.resolve();
-        }
-
         dispatch({ type: LIB_WILL_INIT });
 
+        const initPromise = JitsiMeetJS.init({
+            enableAnalyticsLogging: isAnalyticsEnabled(getState),
+            ...config
+        });
+
+        dispatch({
+            type: LIB_INIT_PROMISE_CREATED,
```

Comment:
I wonder if it wouldn't make sense to call `.init` **before** dispatching WILL_INIT, and have the promise be part of that action instead. WDYT? Do you need any problems with that? Semantically, it does makes sense to me: "lib jitsi meet will initialize, here is the promise".

Reply:
@virtuacoplenny had the same idea and I replied in the previous comments but since the situation changed:
I like the idea to use existing action for the promise but we need an action before the init. some error handlers and logging initializations are done before JitsiMeetJS.init(). Maybe in future we can improve this somehow and reuse the WILL_INIT action but I think at this point this would be hard and will involve refactoring that will be out of scope of the PR.

Reply:
But why do we need the action **before** init?